uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -46,6 +47,31 @@ var Promise = require('es6-promise').Promise;
     });
   });
 
+  /**
+   * Gets a map of contact roles and IDs
```

Comment:
not really descriptive of what's going on here

Reply:
Done.